microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/security/ListenerSecurityConfigurationTest.java

```diff
@@ -215,13 +217,23 @@ private static void enableHTTPListener() throws InterruptedException, TimeoutExc
                         "socket-binding=http,redirect-socket=https,enable-http2=true)");
         result.assertSuccess();
 
-        result = listenersConfigurationOnlineManagementClient.execute(
-                "/subsystem=remoting/http-connector=http-remoting-connector:write-attribute(name=connector-ref, value=default)");
-        result.assertSuccess();
+        if (isHttpRemotingConnectorSet()) {
+            result = listenersConfigurationOnlineManagementClient.execute(
+                    "/subsystem=remoting/http-connector=http-remoting-connector:write-attribute(name=connector-ref, value=default)");
+            result.assertSuccess();
+        }
 
         new Administration(listenersConfigurationOnlineManagementClient).reload();
     }
 
+    private static boolean isHttpRemotingConnectorSet()
+            throws InterruptedException, TimeoutException, IOException, CliException {
+
+        ModelNodeResult result = listenersConfigurationOnlineManagementClient.execute(
+                "/subsystem=remoting/http-connector=http-remoting-connector:read-resource()");
+        return result.isSuccess();
```

Comment:
Is there a reason for using a variable at all?

Reply:
I perceive this notation as more comprehensible than a chain of methods. I can remove the variable if you perceive it the other way.

Reply:
Thank you for an explanation, I'm OK with this.